tests/src/PatternAssertion/FilePatternAssert.php

```diff
@@ -53,7 +54,7 @@ protected function assertBaseElements(string $html, string $variant): void {
    *   The crawler.
    */
   protected function assertFile(array $expected, Crawler $crawler): void {
-    $file = $crawler->filter('body > div.mt-4 > div.border.rounded > div:first-child');
+    $file = $crawler->filter('body > .bcl-file-container > .bcl-file.border.rounded > div:first-child');
```

Comment:
The change with the wrapper selector is good, although I think the ticket is asking to remove this direct child selector to allow the use of this assert in the whitelabel person test like content banner assert here: https://github.com/openeuropa/oe_whitelabel/blob/cc8f899dd188e24d3e4b4c83a15012339282c4a1/tests/src/Functional/PublicationContentRenderTest.php#L46

Can you check, please?

Reply:
you are right. I removed the direct body > selectors.